gstd/gstd_pipeline.c

```diff
@@ -159,6 +171,22 @@ gstd_pipeline_class_init (GstdPipelineClass * klass)
       "The event handler of the pipeline",
       GSTD_TYPE_EVENT_HANDLER, G_PARAM_READABLE | G_PARAM_STATIC_STRINGS);
 
+   properties[PROP_POSITION] =
+      g_param_spec_int64 ("position", "Position",
+      "The query position of the pipeline",
+      0, /* Min value */
+      G_MAXINT64,
+      0, /* Default value */
+      G_PARAM_READABLE | G_PARAM_STATIC_STRINGS);
+
+    properties[PROP_DURATION] =
+      g_param_spec_int64 ("duration", "Duration",
+      "The duration of the media stream pipeline",
+      0, /* Min value */
+      G_MAXINT64,
+      0, /* Default value */
```

Comment:
This is not portable, this will fail in imx6, por example. Use G_GINT64_CONSTANT(0) instead.